spec/rails_helper.rb

```diff
@@ -0,0 +1,29 @@
+require "simplecov"
+SimpleCov.start "rails" do
+  enable_coverage :branch
+end
+
+require "spec_helper"
+
+ENV["RAILS_ENV"] = "test"
+require File.expand_path("../config/environment", __dir__)
+require "rspec/rails"
+require "capybara/rails"
+require "webmock/rspec"
+require "slimmer/rspec"
+WebMock.disable_net_connect!(allow_localhost: true)
+require "timecop"
+
+Dir[Rails.root.join("spec/support/*.rb")].sort.each { |f| require f }
+
+GovukTest.configure
+
+RSpec.configure do |config|
+  config.include ContentStoreHelpers, type: :controller
+
+  config.include Helpers::Components, type: :feature
+  config.include ContentStoreHelpers, type: :feature
+  config.include FeatureHelpers, type: :feature
+
+  config.include Helpers::Components, type: :view
+end
```

Comment:
I think we don't need this file right? If we put all the config into the new spec_helper file (added in the **Install RSpec and init** commit) then I think we can lose all of the `require "rails_helper"` statements at the top of all the specs

Reply:
Hmm - the general rule AFAIK is that you want separate spec_helper and rails_helper files in case you are developing locally and want to run a subset of your tests (in which case you only include rails_helper stuff in the files that actually need rails_helper). Maybe we don't run subsets of the test suite locally enough to justify that, though? It makes no difference to CI.

Reply:
There's no rails helper in collections. What else uses RSpec?

Reply:
Hmm, I was going to say imminence has a rails_helper but no spec_helper, but I did that conversion from minitest too, so that could just be my mistake.

Reply:
I've converted it here to spec_helper and removed the includes. Seems like a good idea, and easy enough to put them back if necessary.